blogs/whats_new_1.7/index.md

```diff
@@ -0,0 +1,153 @@
+    ebpf:
+      flowFilter:
+        enable: true
+        tcpFlags: SYN
+      sampling: 1
+```
+<p style="text-align: center">Figure 5: eBPF flow filter for TCP SYN and UDP traffic</p>
+
+In the Traffic flows table, you can add the **TCP flags** column.  You can also filter on this column (Figure 6).
+
+![Traffic flows - TCP flags](images/traffic_flows-tcp_flags.png)
+<div style="text-align: center">Figure 6: Traffic flows - TCP flags</div>
+
+
+## Network Observability in Developer perspective
+
+For the first time, NetObserv is providing observability in the Developer perspective!  If you are in Administrator perspective, click **Administrator** in the upper left corner and select **Developer**.  Click **Project** and select a project in the drop down near the top of the page.  In the **Inventory** section, click **Pods** or **Service**.   Click a particular item on a row.  There will be a **Network Traffic** tab.  Click this link to bring up the familiar page with *Overview*, *Traffic flows*, and *Topology* tabs filtered to this pod or service (Figure 7).
+
+![Developer perspective - Network Observability](images/developer_perspective-network_traffic.png)
+<div style="text-align: center">Figure 7: Network Observability in Developer perspective</div>
+
+
+## Security features
+
+There are two security-related features in this release.
+
+### FIPS compliance
+
+Network Observability 1.7 is compliant with FIPS 140-3, ensuring that it meets security requirements for cryptographic modules.  This means it can be used when OpenShift is in [FIPS mode](https://docs.openshift.com/container-platform/4.17/installing/overview/installing-fips.html).
+
+### Lock down access to Network Observability
+
+For added security, it is simple to lock down access to Network Observability so only known components can access it.  When you create the FlowCollector resource, enable this feature (Figure 8).  This creates a Kubernetes network policy in the *netobserv* and *netobserv_privileged* namespaces to restrict its access.  Although this is disabled by default, it is recommended to enable this or create your own network policies.
+
+![FlowCollector - Network policy](images/flowcollector-network_policy.png)
+<div style="text-align: center">Figure 8: Network policy</div>
+
+
+## Network Observability CLI enhancements
+
+Network Observability CLI (NetObserv CLI) is a kubectl plug-in that installs a separate Network Observability from the command line and displays a text-based flows table and can also capture packets to a pcap file.  It uses no storage!  When you exit the program, it cleans up and removes itself.
+
+You don't need to install the Network Observability Operator because NetObserv CLI will do it for you.  Just [follow the instructions here](https://docs.openshift.com/container-platform/4.17/observability/network_observability/netobserv_cli/netobserv-cli-install.html) to install NetObserv CLI.  The prerequisite is to have `oc` installed on your computer.
```

Comment:
Technically speaking the CLI is not installing the operator.

```suggestion
You don't need to install the Network Observability Operator because NetObserv CLI will handle the necessary components for you.  Just [follow the instructions here](https://docs.openshift.com/container-platform/4.17/observability/network_observability/netobserv_cli/netobserv-cli-install.html) to install NetObserv CLI.  The prerequisite is to have `oc` installed on your computer and be cluster admin.
```